Automation/SQL/azureDeploy.parameters.json

```diff
@@ -0,0 +1,16 @@
+
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+      "serverName": {
+          "value": "GEN-UNIQUE"
```

Comment:
These would all be input parameters - don't set the values here. You can have a "defaultValue" though

Reply:
I just deleted that file :)